Frameworks/Foundation/NSProxy.mm

```diff
@@ -202,12 +210,40 @@ - (BOOL)conformsToProtocol:(Protocol*)aProtocol {
     return class_conformsToProtocol(forwardClass, aProtocol);
 }
 
+// NOTE: long return value to allow nonfatal continuation to get a "valid" result (for non-fpret/non-stret calls)
+static long _throwUnrecognizedSelectorException(id self, Class isa, SEL sel) {
```

Comment:
why is this helper needed if it's only being used in -[doesNotRecognizeSelector] ?